src/content/docs/en/reference/publish-to-npm.mdx

```diff
@@ -39,7 +39,7 @@ Before diving in, it will help to have a basic understanding of:
 :::
 
 
-To create a new package we strongly recommend configuring your development environment to use **workspaces** within your project. This will allow you to develop your component alongside a working copy of Astro.
+To create a new package, you should configure your development environment to use **workspaces** within your project. This will allow you to develop your component alongside a working copy of Astro.
```

Comment:
```suggestion
To create a new package, configure your development environment to use **workspaces** within your project. This will allow you to develop your component alongside a working copy of Astro.
```
We don't use "should" in the docs because it can be confusing advice. It *sounds* like you "must" (in which case, we write "must" or something else similarly absolute), but it doesn't *mean* that you must.

I'm OK making this an absolute statement here as above. Note: the original was actually fine because the "we" being used was "Astro / Docs people". We don't use "we" when writing instructions for someone *else* to follow, because "we" are not there. But, it's perfectly fine to say "we" when you don't include the reader, and are instead saying what we at Astro recommend!